etc/start_faux

```diff
@@ -53,5 +53,10 @@ if [ -n "${options[dnsmasq]}" ]; then
     dnsmasq --log-facility=/tmp/dnsmasq.log --log-dhcp
 fi
 
+if [ -n "${options[http]}" ]; then
+    echo Listening on TCP 80...
+    python3 -m http.server &
```

Comment:
This by default listens on port 8000.

Reply:
Updated the echo message. I saw daq does fail the device even if it is listing on 8000.